aw_transform/sort_by.py

```diff
@@ -15,3 +15,9 @@ def sort_by_duration(events) -> List[Event]:
 
 def limit_events(events, count) -> List[Event]:
     return events[:count]
+
+def sum_durations(events) -> timedelta:
+    s = timedelta()
+    for event in events:
+        s += event.duration
```

Comment:
Could be done simply like: `return sum(event.duration for event in events) if events else timedelta()`

Reply:
Since the function should always return a timedelta I wrote:

timedelta(seconds=(sum(event.duration.total_seconds() for event in events)))